app/frontend/components/domains/requirement-template/screens/jurisdiction-edit-digital-permit-screen/index.tsx

```diff
@@ -83,6 +92,36 @@ export const JurisdictionEditDigitalPermitScreen = observer(function Jurisdictio
     reset(formFormDefaults(jurisdictionTemplateVersionCustomization))
   }, [jurisdictionTemplateVersionCustomization?.customizations])
 
+  const location = useLocation()
+  const queryParams = new URLSearchParams(location.search)
+  const isCompare = queryParams.get("compare") === "true"
```

Comment:
since we are using react router, please use their apis when possible https://reactrouter.com/en/main/hooks/use-search-params